kafka-connect/src/main/scala/net/manub/embeddedkafka/connect/EmbeddedConnectConfig.scala

```diff
@@ -0,0 +1,46 @@
+package net.manub.embeddedkafka.connect
+
+import java.io.File
+
+import net.manub.embeddedkafka.EmbeddedKafkaConfig
+
+import org.apache.kafka.connect.runtime.standalone.StandaloneConfig
+import org.apache.kafka.connect.runtime.WorkerConfig
+
+/**
+  * Kafka Connect config generator.
+  *
+  * @tparam C an [[EmbeddedKafkaConfig]]
+  */
+private[embeddedkafka] trait EmbeddedConnectConfig[C <: EmbeddedKafkaConfig] {
+  protected[embeddedkafka] def baseConnectConfig(
+      connectPort: Int,
+      offsets: File
```

Comment:
I'd use `java.nio.file.Path` instead, as it's already used by the library sometimes; also it works nicely with `Files.createTempFile` ;)

Reply:
Done!